src/libs/transforms.ts

```diff
@@ -65,48 +65,56 @@ export interface FileRecord {
   additionalComms: string;
 }
 
+export function parseZuoraDataFile(file: string): string[][] {
+  const records: string[][] = parse(file);
+  records.shift(); // getting rid of the first record containing the columns descriptions
+  return records;
+}
+
 export function subscriptionsDataFileToSubscriptions(file: string): ZuoraSubscription[] {
-  const splitLines = str => str.split(/\r?\n/);
-  const lines = splitLines(file);
-  lines.shift();
-  const subscriptions = lines.map((line) => {
-    const elements = line.split(",");
+  const records = parseZuoraDataFile(file);
+  const subscriptions = records.map((record) => {
     return {
-      subscription_name: elements[0],
-      subscription_delivery_agent: elements[1],
-      sold_to_address1: elements[2],
-      sold_to_address2: elements[3],
-      sold_to_city: elements[4],
-      sold_to_postal_code: elements[5],
-      sold_to_first_name: elements[6],
-      sold_to_last_name: elements[7],
-      sold_to_special_delivery_instructions: elements[8],
-      quantity: elements[9]
+      subscription_name: record[0],
+      subscription_delivery_agent: record[1],
+      sold_to_address1: record[2],
+      sold_to_address2: record[3],
+      sold_to_city: record[4],
+      sold_to_postal_code: record[5],
+      sold_to_first_name: record[6],
+      sold_to_last_name: record[7],
+      sold_to_special_delivery_instructions: record[8],
+      quantity: record[9]
     }
   });
   return subscriptions;
 }
 
 export function holidayNamesDataFileToNames(file: string): string[] {
-  const splitLines = str => str.split(/\r?\n/);
+  const splitLines = (str: string) => str.split(/\r?\n/);
```

Comment:
shouldn't this be using the csv parser as well?  If it's one column that we know the format it should be ok, but if we've got it, why not keep it (in case someone adds a new column later)

Reply:
Good question. That query doesn't because we are only querying one field, subscription names